### DISCUSSION TITLE
Support for Kinyarwanda?

### DISCUSSION THREAD

**Discussion by desiregasagara** [openai/whisper#2425] [General]
Hi there, I was looking to use whisper to translate my grandparent's advice in an interview they put on youtube and noticed that unfortunately this isn't yet available. Can this be supported in future?

**Comment by glangford** [comment]
If it is on YouTube, try the YT auto-translate to English feature (turn on closed captions, click the gear button, click Subtitles/CC, click Auto translate)

If this isn't what you are looking for,  Kinyarwanda is in the list of supported languages for Google Cloud:

https://cloud.google.com/speech-to-text/docs/speech-to-text-supported-languages

See also Lanfrica
https://lanfrica.com/record/kinyarwanda-tts-model#:~:text=Lanfrica%20%7C%20Kinyarwanda%20TTS%20Model
